index.html

```diff
@@ -5435,22 +5440,33 @@ <h2 id="AudioWorkletNode">
               </p>
             </dd>
             <dt>
-              attribute EventHandler onmessage
+              readonly attribute AudioParamMap parameters
             </dt>
             <dd>
               <p>
-                The <code>onmessage</code> handler is called whenever the
-                associated <a><code>AudioWorkletProcessor</code></a> posts a
-                message back to the main thread.
+                The <code>parameters</code> attribute is a collection of
+                <a>AudioParam</a> objects with associated names. This maplike
+                object is populated from a list of <a>AudioParamDescriptor</a>s
+                in the <a>AudioWorkletProcessor</a> class definition at the
+                instantiation.
               </p>
             </dd>
             <dt>
-              void postMessage(any message, optional sequence&lt;object&gt;
-              transfer)
+              attribute EventHandler ondata
             </dt>
             <dd>
               <p>
-                <code>postMessage</code> may be called to send a message to the
+                The <code>ondata</code> handler is called whenever the
+                associated <a>AudioWorkletProcessor</a> sends data back to the
+                AudioWorkletNode instance.
+              </p>
+            </dd>
+            <dt>
+              void sendData(any data)
```

Comment:
Perhaps add a description for the `data` parameter. Right now, there's none included.